examples/spring-streams-example/src/main/kotlin/stove/spring/streams/example/kafka/CustomProductionExceptionHandler.kt

```diff
@@ -0,0 +1,25 @@
+package stove.spring.streams.example.kafka
+
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.apache.kafka.streams.errors.ProductionExceptionHandler
+import org.slf4j.LoggerFactory
+import java.lang.Exception
+
+class CustomProductionExceptionHandler : ProductionExceptionHandler {
+  companion object {
+    private val logger = LoggerFactory.getLogger(CustomProductionExceptionHandler::class.java)
+  }
+
+  override fun handle(
+    record: ProducerRecord<ByteArray, ByteArray>?,
```

Comment:
They are not nullable actually, you can remove the nullability. Both from exception and record.

Reply:
done